joblib/numpy_pickle.py

```diff
@@ -89,7 +89,7 @@ def write_array(self, array, pickler):
                                                   'zerosize_ok'],
                                            buffersize=buffersize,
                                            order=self.order):
-                pickler.file_handle.write(chunk.tostring('C'))
+                pickler.file_handle.write(chunk.data)
```

Comment:
I am just starting to look at this. It looks great. I'd like to tweek a
bit the figure, could you send we the CSV files please.